lib/rules/require-aligned-multiline-params.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * Enforces indentation of parameters in multiline functions
+ *
+ * Types: `Boolean`, `String`, `Number`
+ *
+ * Values:
+ *  - `true` to require parameters are aligned with the body of the function
+ *  - `'firstParam'` to require parameters to be aligned with the first parameter
+ *  - `Number` for the number of columns the parameters should be indented past the function body
+ *
+ * #### Example
+ *
+ * ```js
+ * "disallowSpaceBeforeComma": true
+ * ```
+ *
+ * ##### Valid for `true`
+ *
+ * ```js
+ * var test = function(one, two,
+ *   three, four, five,
+ *   six, seven, eight) {
+ *   console.log(a);
+ * };
+ * ```
+ *
+ * ##### Valid for `2`
+ *
+ * ```js
+ * var test = function(one, two,
+ *     three, four, five,
+ *     six, seven, eight) {
+ *   console.log(a);
+ * };
+ * ```
+ *
+ * ##### Valid for `'firstParam'`
+ *
+ * ```js
+ * var test = function(one, two,
+ *                     three, four, five,
+ *                     six, seven, eight) {
+ *   console.log(a);
+ * };
+ * ```
+ *
+ */
+
+var assert = require('assert');
+
+module.exports = function() {
+};
+
+module.exports.prototype = {
+
+    configure: function(option) {
+        if (typeof option === 'number') {
```

Comment:
In your case, close to none, since you already wrote the logic and tests for it, for supporting it, there some benefit and for developers that could use this as an example for the new rules